packages/lighthouse-service/.env.example

```diff
@@ -0,0 +1,18 @@
+## Uncomment the lines as needed
```

Comment:
Does this microservice not use any environment variables? If so, then you can just delete this file.

But I'm pretty sure it would need some variables for the lighthouse server url, also the database connections. Can you add those variables to this example file?

Reply:
This is using the env varialbles like gitlab url, lighthouse server url and all

Reply:
So can you add those variables to the `.env.example` file? And don't resolve the conversations before it's verified.

Reply:
Updated the env vars